meson.build

```diff
@@ -3,7 +3,6 @@
 project('qrtr',
         'c',
         license : [ 'BSD-3-Clause'],
-        meson_version : '>= 0.64.0', # for feature user options
```

Comment:
well it's useful because you use features that are not available on older meson versions

Reply:
idk.. we have old debian in CI and it goes through